doc/development/submitting/submitting_c.md

```diff
@@ -344,33 +344,19 @@ Notes:
 
 These index manual pages are autogenerated during the build process of GRASS GIS.
 
-## Indentation
+## Source Code Formatting
 
-To promote a consistent coding style for whitespace (spaces, tabs, newlines),
-please use the `indent` tool on all new C files using the following switches:
+C and C++ code is formatted with [ClangFormat](https://clang.llvm.org/docs/ClangFormat.html).
+Contributions to main branch (and grass8 branch for grass-addons) are expected
+to be formatted with `clang-format` (currently with version 15+). The most
+convenient method to install clang-format and format files is [using pre-commit](./submitting.md#use-pre-commit).
+
+Alternatively, using separately installed clang-format on modified files:
 
 ```bash
--bad -bap -bbb -br -bli0 -bls -cli0 -ncs -fc1 -hnl -i4
--nbbo -nbc -nbfda -nbfde -ncdb -ncdw -nce -nfca -npcs -nprs
--npsl -nsc -nsob -saf -sai -saw -sbi0 -ss --no-tabs
+clang-format -i <new_or_modified_file.c>
```

Comment:
Perhaps add a note from where the `clang-format` settings come? (point to our [file in GH](https://github.com/OSGeo/grass/blob/main/.clang-format) or so?)

Reply:
Good point, added.